### PR TITLE
Rtc261081 filelocking arquillian

### PR DESCRIPTION
**Pull request by benjamin-confino**

### CODE REVIEW

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
#build

File: dev/com.ibm.ws.microprofile.faulttolerance.2.0_fat_tck/publish/servers/FaultTolerance20TCKServer/jvm.options [hunk 1]
```diff
@@ -1 +1,2 @@
 -Dcom.ibm.tools.attach.enable=yes
+-Xjit:{org/eclipse/microprofile/config/ConfigProvider.getConfig(Ljava/lang/ClassLoader;)Lorg/eclipse/microprofile/config/Config;}(tracefull,traceInlining,traceCG,log=getConfig.trace)
```
Comment (Azquelt) [inline dev/com.ibm.ws.microprofile.faulttolerance.2.0_fat_tck/publish/servers/FaultTolerance20TCKServer/jvm.options]: We don't want this change. This was just for when I was running personal builds to diagnose a problem on OpenJ9 12.

File: dev/com.ibm.ws.microprofile.metrics.1.1.noAuth_fat_tck/publish/tckRunner/tck/pom.xml [hunk 2]
```diff
@@ -75,7 +75,7 @@
         <dependency>
             <groupId>io.openliberty.arquillian</groupId>
             <artifactId>arquillian-liberty-managed</artifactId>
-            <version>1.0.5</version>
+            <version>1.0.5-anrouse-SNAPSHOT</version>
```
Comment (Azquelt) [inline dev/com.ibm.ws.microprofile.metrics.1.1.noAuth_fat_tck/publish/tckRunner/tck/pom.xml]: This should not be the snapshot version

**Comment by benjamin-confino** [comment]
#build

**Comment by benjamin-confino** [comment]
#build

**Pull request merged**